x/campaign/keeper/msg_server_create_campaign_test.go

```diff
@@ -78,9 +115,20 @@ func TestMsgCreateCampaign(t *testing.T) {
 				CampaignName: sample.CampaignName(),
 				Coordinator:  coordAddr1,
 				TotalSupply:  sample.CoinsWithRange(10, 20),
+				Metadata:     sample.Metadata(20),
 			},
 			err: types.ErrInvalidTotalSupply,
 		},
+		{
+			name: "insufficient balance to cover creation fee",
+			msg: types.MsgCreateCampaign{
+				CampaignName: sample.CampaignName(),
+				Coordinator:  coordAddr1,
```

Comment:
We reuse `coordAddr1` here used above. Does it fails because `coordAddr1` has only enough funds for 1 message?
I would prefer creating a coordinator without fund specifically for this test case to make it more maintainable
For example, the test above `"create a campaign with an invalid token supply"` fails because `TotalSupply` is invalid but it would also fail because of insufficient funds. I would consider giving to `coordAddr1` enough funds for all other test cases